py/desitarget/cuts.py

```diff
@@ -1036,7 +1036,7 @@ def notinBGS_mask(gnobs=None, rnobs=None, znobs=None, primary=None,
     bgs &= (gfluxivar > 0) & (rfluxivar > 0) & (zfluxivar > 0)
 
     # ADM geometric masking cuts from the Legacy Surveys.
-    bgs &= imaging_mask(maskbits, ["BRIGHT", "CLUSTER"])
+    bgs &= imaging_mask(maskbits)
```

Comment:
Are you sure that you want to default to the standard set of cuts on `MASKBITS`, here? Those are `["BRIGHT", "CLUSTER", "GALAXY"]`. From conversations I'd had with @moustakas I thought that you wouldn't want to mask on `GALAXY` so that you could investigate the genuine galaxies within the SGA masks. My understanding could certainly be outdated, though!

Reply:
Good catch @geordie666, thank you. We should *definitely* not be applying the `GALAXY` mask for BGS selection (definitely not for SV and probably not for the main survey, either). 

Another selection tidbit is that `CLUSTER` also throws out the Fornax and Sculptor local-group dwarfs, but these will probably never be BGS targets so I think we're OK.

Reply:
Yes, that's the idea, and for SV we don't mask around large galaxies. However, this cuts are for nominal and we have not decided yet the final DR9 selection. Most probably that we will end up not masking those, but we can come with other changes for dr9 selection so with this change I'm just going back to dr8 selection.

Reply:
Ah, I see, this change is just for the Main Survey selection. I guess we just have to stay on top of the issue of what to do about `MASKBITS` for BGS for the Main Survey.